src/docs/install.md

```diff
@@ -23,19 +23,29 @@ in the root (ie, "buildroot") of your repo:
     :::bash
     curl -L -O https://pantsbuild.github.io/setup/pants && chmod +x pants && touch pants.ini
 
+Before running the new `./pants` script, you should pin the Python version you want to run Pants with.
+To do this, add an entry like so to `pants.ini`, choosing between `2.7`, `3.6`, or `3.7`.
+
+    :::ini
+    [GLOBAL]
+    pants_engine_python_version: 3.6
+
+If you'd like to change which Python version Pants uses, just edit the entry in `pants.ini`.
+
 The first time you run the new `./pants` script it will install the latest version of pants (using
 virtualenv) and then run it.  It's recommended though, that you pin the version of pants.  To do
 this, first find out the version of pants you just installed:
 
     :::bash
     ./pants -V
-    1.0.0
+    1.14.0
 
 Then add an entry like so to `pants.ini` with that version:
 
     :::ini
     [GLOBAL]
-    pants_version: 1.0.0
+    pants_engine_python_version: 3.6
```

Comment:
Question #2: I want to add these notes still, and am not sure where to put them:
```
Some notes when choosing an interpreter:
- This does not mean your Python code has to use this same Python version. See https://www.pantsbuild.org/python_readme.html#configure-the-python-version for how to configure your code's compatibility.
- Ensure you have the pinned Python version accessible on your PATH everywhere Pants is used in your organization.
- Pants will require Python 3.6+ to run as of release 1.17.0.
```

If we move the instruction on pinning lower in the file, past pinning the version, then we can keep those two notes together. Otherwise, is there some sort of gray text box or something I can add that shows it's an aside? I don't want to break the flow but do want to give additional information.